src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -168,6 +168,10 @@
         (.close conn)
         (throw e)))))
 
+(defmethod connection-with-timezone :sql-jdbc
+  [driver database ^String timezone-id]
+  (default-connection-with-timezone driver database timezone-id))
```

Comment:
why was this extracted into a function?

Reply:
To be honest, I have no idea anymore.  I'll back this out.

Reply:
I suspect originally it was so the H2 implementation could call it as well, but that's not needed in the latest changes.